parm/archive/gdasice.yaml.j2

```
@@ -3,8 +3,8 @@ gdasice:
     name: "GDASICE"
     target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gdasice.tar"
     required:
-        - "{{ COM_ICE_HISTORY | relpath(ROTDIR) }}/{{ head }}ic.nc"
-        {% for fhr in range(FHOUT_ICE, FHMAX+1, FHOUT_ICE) %}
-        - "{{ COM_ICE_HISTORY | relpath(ROTDIR) }}/{{ head }}inst.f{{ '%03d' % fhr }}.nc"
+        - "{{ COMIN_ICE_HISTORY | relpath(ROTDIR) }}/{{ head }}ic.nc"
+        {% for fhr in range(FHOUT_OCNICE, FHMAX+1, FHOUT_OCNICE) %}
```

This should still be `FHOUT_ICE`.  `FHOUT_OCNICE` is not defined.

```suggestion
        {% for fhr in range(FHOUT_ICE, FHMAX+1, FHOUT_ICE) %}
```